src/modules/Auth/components/Registration/Registration.css

```diff
@@ -0,0 +1,223 @@
+.primary-color{
+  background-color: #EF9B28 ;
+}
+
+.floating-form  {
+  display: flex;
+  flex-direction: column;
+  background-color: #314a59d4;
```

Comment:
We need to avoid having fixed colors and use css variables instead, as we will need to implement light/dark mode feature. Having fixed colors will make this very difficult to implement.

Reply:
done